scene/gui/popup_menu.cpp

```diff
@@ -1523,6 +1523,7 @@ PopupMenu::PopupMenu() {
 	set_hide_on_item_selection(true);
 	set_hide_on_checkable_item_selection(true);
 	set_hide_on_multistate_item_selection(false);
+	set_flag(Window::FLAG_NO_FOCUS, true);
```

Comment:
This flag should theoretically fix the popup menu gaining focus, but sometimes it breaks for some reason.

Reply:
I'll do some tests.